Modul1/First modul/First modul/Program.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+
+class FindSolutionOfExersize
+{
+    static void Main()
+    {
+        Console.Write("Enter the number elements in array: ");
+        int n = Convert.ToInt32(Console.ReadLine());
+
+        Random random = new Random();
+        List<int> mainMas = new List<int>();
+
+        for (int i = 0; i < n; i++)
+        {
+            mainMas.Add(random.Next(1, 27));
+        }
+
+        List<int> evenNums = new List<int>();
+        List<int> oddNums = new List<int>();
+
+        foreach (int numer in mainMas)
+        {
+            if (numer % 2 == 0)
+            {
+                evenNums.Add(numer);
+            }
+            else
+            {
+                oddNums.Add(numer);
+            }
+        }
+
+        string bukva = "abcdefghijklmnopqrstuvwxyz";
+
+        List<string> evenBukvy = new List<string>();
+        List<string> oddBukvy = new List<string>();
+
+        foreach (int numer in evenNums)
+        {
+            char litera = bukva[numer - 1];
+            if ("aeidhj".Contains(litera))
+            {
+                evenBukvy.Add(litera.ToString().ToUpper());
+            }
+            else
+            {
+                evenBukvy.Add(litera.ToString());
+            }
+        }
+
+        foreach (int numer in oddNums)
+        {
+            char litera = bukva[numer - 1];
+            if ("aeidhj".Contains(litera))
+            {
+                oddBukvy.Add(litera.ToString().ToUpper());
+            }
+            else
+            {
+                oddBukvy.Add(litera.ToString());
+            }
+        }
```

Comment:
This code block looks like code block from lines 39:50 and can be moved to method